prepare_jumpbox.md

```diff
@@ -16,6 +16,9 @@ RDP into the jumpbox (you can get the IP using AzurePortal). The user and passwo
 
 - Log in the VM again. Open the Microsoft Store, then search WSL, choose Ubuntu, and install it.
```

Comment:
This instruction says "Microsoft Store" -- which will never be the case.  Can we update this?